app/views/shared/_footer_lite.html.slim

```diff
@@ -3,9 +3,9 @@ footer.footer.bg-navy.white
     .clearfix
       .sm-col = t('shared.footer_lite.gsa')
       .sm-col-right.caps
-        = link_to t('links.help'), help_path,
+        = link_to t('links.help'), "https://#{APP_NAME}/help",
```

Comment:
should we use something other than `APP_NAME` here? as-is, this does not work for env-specific links